analyses/survival-analysis/survival-analysis_HGG_DMG.Rmd

```diff
@@ -0,0 +1,210 @@
+---
+title: "Survival Analysis for subtypes of HGG/DMG"
+output: 
+  html_notebook:
+    toc: TRUE
+    toc_float: TRUE
+author: C. Savonen for ALSF CCDL, Krutika Gaonkar for D3B
+date: 2019
+params:
+  plot_ci: TRUE
+---
+
+**Purpose:** 
+
+Runs survival analysis models for subtypes of Diffuse astrocytic and oligodendroglial tumors
+
+## Usage 
+
+Uses a wrapper function (`survival_analysis`) from utils folder. 
+
+## Setup
+
+#### Packages and functions
+
+Read in set up script.
+
+```{r Set up library}
```

Comment:
NOTE: This script does not knit in part to libraries not being used explicitly. Either must load `dplyr` explicitly, or add `dplyr::` to lines where it is relevant. There are some others bugs I note in other comments below.